ebill-product/pom.xml

```diff
@@ -35,6 +35,14 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web-services</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-validation</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-devtools</artifactId>
```

Comment:
No subir dependencia de devtools

Reply:
until we deploy at production we'll maintain this dependency at `pom.xml`